### ISSUE TITLE
YASGUI: add some minimal Documentation

### ISSUE THREAD

**Issue by ingoboerner** [dracor-org/dracor-frontend#45]
Maybe we could come up with some minimal documentation to the SPARQL Endpoint and list some suggested Queries either on the YASGUI-Page itself (as does the ONB-SPARQL Interface https://labs.onb.ac.at/tool/sparql/ – same Implementation, by the way...)
@lehkost ?

**Comment by lehkost** [comment]
Yes, documentation with examples would be great. Maybe add a new repo ```documentation``` and use Markdown, which could then be mirrored to our website via GH pages? We're also trying to find out where best to put our R Notebooks with API query examples, so we might have to have a general discussion within the group. (@mathias-goebel @cmil @Pozdniakov)

**Comment by cmil** [comment]
Generally I would recommend to keep the documentation close to the code, to increase chances it gets updated whenever the code changes. So maybe in this case [dracor-schema](https://github.com/dracor-org/dracor-schema), where the ontology is maintained, would be appropriate?
